nncf/nncf_network.py

```diff
@@ -638,7 +638,7 @@ def get_compression_modules_by_type(self, compression_module_type: ExtraCompress
     def _compression_module_type_to_attr_name(compression_module_type: ExtraCompressionModuleType):
         """Required for backward compatibility with checkpoints that store function and activation
         quantizers directly under corresponding attributes of NNCFNetwork."""
-        if compression_module_type == ExtraCompressionModuleType.ACTIVATION_QUANTIZER:
+        if compression_module_type == ExtraCompressionModuleType.EXTERNAL_QUANTIZER:
             return "activation_quantizers"
```

Comment:
There's still `activation` and `external` at the same time. 
I'd rename everywhere at once. BTW, why previous name is not suitable now?

Reply:
Cannot rename the string representation that easily since it goes into the state dict and, by corollary into the checkpoints. Renamed the entity name in code because the meaning of the underlying storage has changed - it stores quantizer modules that are potentially used both for activations and weights (so the designation "activation quantizers" is not longer applicable), and it does so externally to the underlying model.

Reply:
Adjusted the code to use the new `external_quantizer` naming going forward, and still be able to load the old-style checkpoints (displaying a deprecation warning in this case).